src/wagtail_ai/static_src/AIControl.tsx

```diff
@@ -70,6 +77,7 @@ function ToolbarDropdown({
 }
 
 function AIControl({ getEditorState, onChange }: ControlComponentProps) {
+  const aiPrompts = getAIConfiguration()?.wagtailAiPrompts as Array<Prompt>;
```

Comment:
**Issue:** Why do you need to use type casting (`as`) if wagtailAiPrompts is always an array? Don't use casting unless you have to.